stubs/PyScreeze/pyscreeze/__init__.pyi

```diff
@@ -165,6 +168,7 @@ def locateOnWindow(
     step: int = 1,
     confidence: None = None,
 ) -> Box | None: ...
+def screenshotWindow(title: Unused) -> None: ...  # Not implemented yet.
```

Comment:
I *assume* this is gonna be a string. But it could also be a window handle depending on future implementation. Probably fine as-is since it's accurate, makes stubtest happy, and reduces chance of false-positives later.